examples/io_dat_files.py

```diff
@@ -0,0 +1,53 @@
+"""
+==================================
+Input/Output of dat files for FLCT
+==================================
+
+This example demostrates the use of functions in `sunkit_image.flct`
+to read and write arrays to binary ``dat`` files.
+"""
```

Comment:
Should mention that these dat files are from the original FLCT code and allow new users to open pre existing files.

Reply:
But in this particular example I have not used a pre existing file. But I will add this point in the other one